dali/sasha/saarch.cpp

```diff
@@ -293,19 +299,22 @@ void WUiterate(ISashaCommand *cmd, const char *mask)
             {
                 const char* outputField = outputFields.item(i);
 
-                StringBuffer val;
-                bool found = true;
+                const char *val = nullptr;;
                 if (strieq(outputField, "owner"))
-                    wu->getProp("@submitID", val);
+                    val = wu->queryProp("@submitID");
                 else if (strieq(outputField, "cluster"))
-                    wu->getProp("@clusterName", val);
+                    val = wu->queryProp("@clusterName");
                 else if (strieq(outputField, "jobname"))
-                    wu->getProp("@jobName",val);
+                    val = wu->queryProp("@jobName");
                 else if (strieq(outputField,"state"))
-                    wu->getProp("@state", val);
-                else
-                    found = false;
-                if (found)
+                    val = wu->queryProp("@state");
+                else if (strieq(outputField,"state"))
+                    val = wu->queryProp(dfu ? "Progress/@state" : "@state");
+                else if (strieq(outputField, "command"))
+                    val = wu->queryProp("@command");
+                else if (strieq(outputField, "wuid"))
+                    val = wu->queryName();
```

Comment:
@wangkx - I could see why the above output format options had been removed during the HPCC-11822 refactoring..
do you know why?

Reply:
In the HPCC-11822, I assume that we always want to output wuid (to know which wu information belongs to). The assumption could be wrong.

Reply:
The wuid is a prefix, but "wuid" was an option in the output fields, so removing might break backward compatibility.